tests/sample/ext/drv/sample_ext.vcxproj

```diff
@@ -178,13 +182,13 @@
     </ResourceCompile>
     <ClCompile>
       <AdditionalIncludeDirectories>%(AdditionalIncludeDirectories);$(DDK_INC_PATH);$(SolutionDir)tests\sample\ext\inc;$(SolutionDir)include;$(SolutionDir)libs\platform;$(SolutionDir)libs\platform\kernel;$(OutputPath);$(SolutionDir)external\ebpf-verifier\src;$(SolutionDir)include\kernel</AdditionalIncludeDirectories>
-      <PreprocessorDefinitions>%(PreprocessorDefinitions);BINARY_COMPATIBLE=0;NT;UNICODE;_UNICODE;NDIS60;POOL_NX_OPTIN_AUTO</PreprocessorDefinitions>
+      <PreprocessorDefinitions>%(PreprocessorDefinitions);BINARY_COMPATIBLE=0;NT;UNICODE;_UNICODE;NDIS60;POOL_NX_OPTIN_AUTO;KERNEL_MODE</PreprocessorDefinitions>
```

Comment:
same as earlier: why do we need to define `KERNEL_MODE` here?

Reply:
like above, I think it's more clear, and usable for the future.

Reply:
I think we should not add it if it not being used anywhere. We can always add it in future when it is actually needed.

It also confuses someone reading the code, as they will not find any usage of this.